fs_drift_wrapper/trigger_fs_drift.py

```diff
@@ -130,3 +133,70 @@ def emit_actions(self):
                     interval['95%'] = float(flds[9])
                     interval['99%'] = float(flds[10])
                     yield interval, '-rsptimes'
+
+        # process counter data
+
+        previous_obj = None
+        for fn in os.listdir(rsptime_dir):
+            if fn.startswith('counters') and fn.endswith('json'):
+                pathnm = os.path.join(rsptime_dir, fn)
+                matched = counters_regex_prog.match(fn)
+                thread_id = matched.group(1)
+                with open(pathnm, 'r') as f:
+                    records = [ l.strip() for l in f.readlines() ]
+                json_start = 0
+                for k, l in enumerate(records):
+                    if l == '{':
+                        json_start = k
+                    if l == '}{' or l == '}':
+                        # extract next JSON string from counter logfile
+
+                        json_str = ' '.join(records[json_start:k])
+                        json_str += ' }' 
+                        if l == '}{':
+                            records[k] = '{'
+                        json_start = k
+                        json_obj = json.loads(json_str)
+                        rate_obj = self.compute_rates(json_obj, previous_obj)
+                        previous_obj = json_obj
+
+                        # timestamp this sample
+
+                        time_since_test_start = float(rate_obj['elapsed-time'])
+                        counter_time = time_since_test_start + start_time
+                        timestamp_str = time.strftime('%Y-%m-%dT%H:%M:%S.000Z', time.gmtime(counter_time))
+                        rate_obj['date'] = timestamp_str
+
+                        # add other info needed to display data in elastic search
+
+                        rate_obj['thread'] = thread_id
+                        rate_obj['cluster_name'] = self.cluster_name
+                        rate_obj['user'] = self.user
+                        rate_obj['uuid'] = self.uuid
+                        rate_obj['sample'] = self.sample
+                        yield rate_obj, '-rates-over-time'
```

Comment:
this should `rates-over-time`

Reply:
@bengland2 after the change the starting hyphen needs to be removed

Reply:
fixed